themes/inbeat/assets/js/main.js

```diff
@@ -163,6 +163,35 @@ function testimonial() {
 }
 
 function pricing() {
+    //set toggle 
+    var toggle = document.getElementsByClassName('toggle-container')
```

Comment:
- use `const` or `let` over `var`
- for better consistency, you could use `document.querySelectorAll('.toggle-container')`

Reply:
1. Since this repo is in ES5, we are using `var` only in this whole file. Even `const` / `let` aren't acceptable here.
2. Replaced with `querySelectorAll `